Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -569,14 +621,15 @@ private void saveControlPosition(int sharedPrefsId, int x, int y)
 	 * InputOverlayDrawableButton. Simply add it to the HashSet of overlay items and wait
 	 * for Android to call the onDraw method.
 	 * 
-	 * @param context  The current {@link Context}.
-	 * @param resId    The resource ID of the {@link Drawable} to get the {@link Bitmap} of.
+	 * @param context The current {@link Context}.
```

Comment:
I hate to say it, but this isn't what I meant! I meant that the longest parameter should have its description one space apart, and all the shorter parameters should have more spaces so that their descriptions are aligned with the longest one. For example, in this case "defaultResId" and "pressedResId" are good, but "context" should be spaced to align with them.